public/app/core/components/Login/LoginForm.tsx

```diff
@@ -42,7 +42,7 @@ export const LoginForm = ({ children, onSubmit, isLoggingIn, passwordHint, login
             id={usernameId}
             autoFocus
             autoCapitalize="none"
-            placeholder={loginHint}
+            placeholder={t('login.form.username-placeholder', 'email or username')}
```

Comment:
Both of these should be like

```suggestion
            placeholder={loginHint.length ? loginHint : t('login.form.username-placeholder', 'email or username')}
```